sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -89,11 +89,11 @@ directive:
 - where-operation: AnalyzeConversation_SubmitJob
   transform: |
     var apiVersion = $doc.info.version + "/";
-    $.description = $.description + "\n\nSee https://learn.microsoft.com/rest/api/language/" + apiVersion + "conversation-analysis-runtime/submit-job for more information.";
+    $.description = $.description + "\n\nSee https://learn.microsoft.com/rest/api/language/" + apiVersion + "analyze-conversation/submit-job for more information.";
 - where-operation: ConversationAnalysis_AnalyzeConversation
   transform: |
     var apiVersion = $doc.info.version + "/";
-    $.description = $.description + "\n\nSee https://learn.microsoft.com/rest/api/language/" + apiVersion + "conversation-analysis/analyze-conversation for more information.";
+    $.description = $.description + "\n\nSee https://learn.microsoft.com/rest/api/language/" + apiVersion + "conversation-analysis-runtime/analyze-conversation for more information.";
```

Comment:
These links appear to have changed for GA: https://review.learn.microsoft.com/en-us/rest/api/language/2023-04-01/analyze-conversation?branch=coglanguage202304

Reply:
Yeah. Link checker caught those for me a while back. Does it not run regularly on your repo? Might consider pinging your engsys person about that.

Reply:
These were valid URLs for 2022-10-01-preview. They swapped which operation falls under "runtime" for this GA version (sync is "runtime", async is just "analyze-conversation"). Do you need to make the same update? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md?plain=1#L171

That being said, we do run a link checker in our CI, but it's possible it doesn't run on src code. I'll have to ask.

Reply:
I'll have to take a look, but our link checker hasn't flagged them. Maybe something with it changed. I'll click on each of our URLs and make sure they are valid myself.